css/index.css

```diff
@@ -124,7 +124,7 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1 {
```

Comment:
Since you gave your elements 2 different classes, you no longer need to repeat styles for each box. All of your boxes will be styled using the .boxes class and each box will be styled with the color using their individual styling name.